x-pack/platform/plugins/private/upgrade_assistant/server/routes/reindex_data_streams/reindex_data_stream.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { errors } from '@elastic/elasticsearch';
+import { i18n } from '@kbn/i18n';
+
+import { error } from '../../lib/data_streams/error';
+import { API_BASE_PATH } from '../../../common/constants';
+import { DataStreamReindexStatusResponse } from '../../../common/types';
+import { versionCheckHandlerWrapper } from '../../lib/es_version_precheck';
+import { dataStreamReindexServiceFactory } from '../../lib/data_streams';
+
+import { RouteDependencies } from '../../types';
+import { mapAnyErrorToKibanaHttpResponse } from './map_any_error_to_kibana_http_response';
+
+export function registerReindexDataStreamRoutes({
+  router,
+  licensing,
+  log,
+  getSecurityPlugin,
+  lib: { handleEsError },
+}: RouteDependencies) {
+  const BASE_PATH = `${API_BASE_PATH}/data_streams_reindex`;
```

Comment:
```suggestion
  const BASE_PATH = `${API_BASE_PATH}/reindex_data_streams`;
```

very much nit; feel free to ignore, just seems a little clearer